src/ResourceManagement/Insights/Microsoft.Azure.Insights/Generated/Management/Insights/IIncidentsOperations.cs

```diff
@@ -1,45 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
There is no PowerShell or Node.js for this API, nobody is using it. As a first step to deprecate it I put the two operations related to alert rule incidents in a separate file and renamed the API as AlertRuleIncidents so that the documentation team does not publish any documentation for it as it was before.